src/main/java/Events.java

```diff
@@ -0,0 +1,14 @@
+public class Events extends UserContent{
+    protected String at;
+
+    public Events(String description, String at, int listIndex) {
+        super(description, listIndex);
+        setTaskName("E");
+        this.at = at;
+    }
+
+    @Override
```

Comment:
The optional @Override makes it clear for the reader to know there is a modification to the method from parent.